sale_blanket_order/report/templates.xml

```diff
@@ -0,0 +1,180 @@
+                                    <td class="text-right">
+                                        <span t-field="l.price_unit" />
+                                    </td>
+                                    <td class="text-center">
+                                        <span t-field="l.date_schedule" />
+                                    </td>
+                                    <td class="text-right">
+                                        <span t-field="l.original_uom_qty" />
+                                        <span
+                                            t-field="l.product_uom"
+                                            groups="uom.group_uom"
+                                        />
+                                    </td>
+                                    <td class="text-right">
+                                        <span
+                                            t-field="l.price_subtotal"
+                                            t-options='{"widget": "monetary", "display_currency": l.currency_id}'
+                                        />
+                                    </td>
+                                </t>
+                                <t t-if="l.display_type == 'line_section'">
+                                    <td name="td_section_line" colspan="99">
+                                        <span t-field="l.name" />
+                                    </td>
+                                    <t t-set="current_section" t-value="l" />
+                                    <t t-set="current_subtotal" t-value="0" />
+                                </t>
+                                <t t-if="l.display_type == 'line_note'">
+                                    <td name="td_note_line" colspan="99">
+                                        <span t-field="l.name" />
+                                    </td>
+                                </t>
+                            </tr>
+                            <t
+                                t-if="current_section and (l_last or doc.line_ids[l_index+1].display_type == 'line_section')"
+                            >
+                                <tr class="is-subtotal text-right">
+                                    <td name="td_section_subtotal" colspan="99">
+                                        <strong class="mr16">Subtotal</strong>
+                                        <span
+                                            t-esc="current_subtotal"
+                                            t-options='{"widget": "monetary", "display_currency": doc.pricelist_id.currency_id}'
+                                        />
+                                    </td>
+                                </tr>
+                            </t>
+                        </t>
+                    </tbody>
+                </table>
+                <div class="clearfix">
+                <div class="row">
+                <div
```

Comment:
Change ml-auto to ms-auto if you want the clearfix to be aligned on the right side of the doc